zuul-core/src/test/java/com/netflix/zuul/message/http/CookieSanitizerTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License");
+ *      you may not use this file except in compliance with the License.
+ *      You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *      Unless required by applicable law or agreed to in writing, software
+ *      distributed under the License is distributed on an "AS IS" BASIS,
+ *      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *      See the License for the specific language governing permissions and
+ *      limitations under the License.
+ */
+package com.netflix.zuul.message.http;
+
+import static com.netflix.zuul.message.http.CookiesSanitizer.cleanCookieHeader;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CookieSanitizerTest {
+
+    @Test
```

Comment:
Can we add a test here that also verifies that the class parses the additional patterns injected via FP correctly, ie. logic in the static initialization block?